rubrik_cdm/cluster.py

```diff
@@ -20,9 +20,76 @@
 
 
 class Cluster(Api):
-    """This class contains methods related to the managment of the Rubrik cluster itself.
+    """This class contains methods related to the management of the Rubrik cluster itself.
     """
 
+    def configure_replication_private(self, username, password, target_ip,
+                              ca_certificate=None, timeout=30):
+        """Configure replication partner as specified by user using PRIVATE NETWORK (direct connection)
```

Comment:
docstring should use the `docBlockr` format. Take a look at the other functions for examples